spec/models/manageiq/providers/autosde/storage_manager/refresher_spec.rb

```diff
@@ -135,7 +135,7 @@
 
         assert_inventory_not_changed { run_targeted_refresh(ems.cloud_volumes.find_by(:ems_ref => "ac287c2d-1776-48a3-a5c9-06327f4a57c4")) }
       end
-
+      z
```

Comment:
```suggestion
```

Reply:
Sorry, I was trying to resolve the tests' problems and apparently because of the ctrl+z...

Anyway, I'm having a hard time solving them (specially when I see that everything works fine). I understand that the data it's not really doing an API call so I'm trying to solve this from the cassettes, although I don't see anywhere inside them "new-volume".

Reply:
@Lauferin it looks like the issue is just updating the tests to expect a name on the cloud volume rather than `nil` as it does currently